knowledge_graph/wikibase.py

```diff
@@ -87,7 +87,7 @@ class WikibaseSession:
     )
     classifier_id_property_id = os.getenv("WIKIBASE_CLASSIFIER_ID_PROPERTY_ID", "P20")
 
-    def __init__(
+    async def __init__(
```

Comment:
if I remember correctly, python can't handle async inits, so this will always fail!